components/brave_adaptive_captcha/buildflags/buildflags.gni

```diff
@@ -1,3 +1,3 @@
 declare_args() {
-  brave_adaptive_captcha_enabled = is_mac || is_linux || is_win || is_ios
+  brave_adaptive_captcha_enabled = true
```

Comment:
Do we still need it?

Reply:
@szilardszaloki we are keeping this flag to be sure based on @emerick suggestion.